### ISSUE TITLE
Clicking on container names does nothing

### ISSUE THREAD

**Issue by joestump** [lllllllillllllillll/DweebUI#95]
The URL is `http://AppLink:AppPort`, but I couldn't find `AppLink` or `AppPort` in the code and they don't appear to be common labels in Docker or anything. Is it possible to set these somehow for containers not deployed through DweebUI?

**Comment by lllllllillllllillll** [comment]
This is fixed in v0.70-dev, which allows you to set a custom IP address or domain name in Settings.

**Comment by lllllllillllllillll** [comment]
@joestump If you're curious about where the code is: 
The container card is views/partials/containerFull.html.
Then in controllers/dashboard.js it replaces "AppLink" with either localhost or the custom link.

**Comment by xuqingkuang** [comment]
> This is fixed in v0.70-dev, which allows you to set a custom IP address or domain name in Settings.

Does it mean that DweebUI could be a service startup program that functions as a service portal?

It's recommended that the `AppLink` could load from the hostname of current URL by default, and the `AppPort` could load from the mapped host port.

**Comment by lllllllillllllillll** [comment]
> Does it mean that DweebUI could be a service startup program that functions as a service portal?

Yeah. I need to expand on options for container cards links. I like the idea of splitting up the link.

bcurran3 (https://github.com/lllllllillllllillll/DweebUI/issues/14) suggested using container labels to generate the links.

**Comment by xuqingkuang** [comment]
Yeah, you may can try the implementation of https://casaos.io/. It's running on my debian.

The CasaOS aims to be a NAS system, but my system doesn't require the harddrive management features, and the container management feature isn't powerful enough.
